tests/behat/atto_userEditsFormulaDeletedDuringEdition.feature

```diff
@@ -61,8 +61,8 @@ Feature: User edits deleted formula
     And I wait until Wirisformula formula exists
     And I dbClick on WirisFormula with alt equals to "1 plus 1"
     And I wait until MathType editor is displayed
-    And I click on WirisFormula with alt equals to "1 plus 1"
-    And I press the enter key
+    And I click on "Page content" field
```

Comment:
Is this selecting the formula or all the content in the editable area?

Reply:
is not selecting anything; it will just click on the edit area, where only the formula is present.
Since the caret is positioned at the end, the next step, pressing the backspace button, will delete it.

The steps before made no sense (press enter?), and the behavior in Chrome vs Firefox was different (one needs a click, the other a double click). This solution works for both.